h5p-bildetema-words-grid-view/src/components/Word/Word.tsx

```diff
@@ -26,7 +26,8 @@ export const Word: React.FC<WordProps> = ({ textVisible, word }) => {
   const nextLabel = useL10n("nextImageLabel");
 
   const renderImages = (): JSX.Element => {
-    const multipleImages = images && images.length > 1;
+    const numberOfImages = images.length;
+    const multipleImages = images && numberOfImages > 1;
```

Comment:
If there's actually a risk of `images` being nullish, this will crash.

```suggestion
    const numberOfImages = images?.length ?? 0;
    const multipleImages = numberOfImages > 1;
```

Reply:
Yes, good catch! Added now